snapcraft/plugins/godeps.py

```diff
@@ -61,13 +70,28 @@ def schema(cls):
         schema['properties']['go-importpath'] = {
             'type': 'string',
         }
+        schema['properties']['go-packages'] = {
+            'type': 'array',
+            'minitems': 1,
+            'uniqueItems': True,
+            'items': {
+                'type': 'string',
+            },
+            'default': [],
+        }
 
         # The import path must be specified.
         schema['required'].append('go-importpath')
 
         return schema
 
     @classmethod
+    def get_build_properties(cls):
+        # Inform Snapcraft of the properties associated with building. If these
+        # change in the YAML Snapcraft will consider the build step dirty.
+        return ['go-packages']
```

Comment:
The `go` plugin grabs these in the `pull` step. Is there a reason we're adding it to the `build` step here? I suspect that'll fail on LP since internet isn't available during `build`.